### DISCUSSION TITLE
Get rid of some default styles

### DISCUSSION THREAD

**Discussion by Luffyyy** [epicmaxco/vuestic-ui#1917] [Ideas]
So there's this style that's being applied by vuestic:

```css
.flex>* {
    flex: 1 1 auto;
}
```
Would be nice to be able to either turn it off, or have this being applied in a different way, I think it just messes up with other UIs that may use flexbox.
I tried reverting it like so in Nuxt 3's app.vue
```html
<style>
.flex > * {
    flex: revert;
}
</style>
```
But it doesn't seem to really help and some elements in my site are still messed up as I use flex a lot.

I don't know if there are more examples of such styles in this framework, but it's possible it's better to have these not applied globally, but by classes provided by vuestic's components.

**Comment by m0ksem** [comment]
@Luffyyy, Thanks! We definitely will fix this issue in scope of grid.scss rework.

In nearby feature. You will be able to remove grid.css from project if you prefer. https://github.com/epicmaxco/vuestic-ui/pull/1801

**Comment by Luffyyy** [comment]
Oh I just realized, the problem is with windicss using .flex as a class name and not necessarily flexbox by itself
